src/openrct2/ride/Vehicle.h

```diff
@@ -468,19 +468,20 @@ namespace OpenRCT2::VehicleFlags
     constexpr uint32_t PoweredCarInReverse = (1 << 3);
     constexpr uint32_t ReadyToDepart = (1 << 4);
     constexpr uint32_t Testing = (1 << 5);
-    constexpr uint32_t CurrentlyColliding = (1 << 6); // When go-karts are colliding, they have a higher chance of changing
-                                                      // lanes
-    constexpr uint32_t StoppedOnLift = (1 << 7);      // Used on rides when safety cutout stops them on a lift
+    // When go-karts are colliding, they have a higher chance of changing lanes
+    constexpr uint32_t CurrentlyColliding = (1 << 6);
+    // Used on vehicles when a safety cut-out stops them, such as RCs on a lift hill and powered rides
+    constexpr uint32_t StoppedBySafetyCutOut = (1 << 7);
     constexpr uint32_t CarIsBroken = (1 << 8);
     constexpr uint32_t TrainIsBroken = (1 << 9);
     constexpr uint32_t StoppedOnHoldingBrake = (1 << 10);
-    constexpr uint32_t CarIsInverted = (1 << 11); // Used on rides where trains can run for extended periods of time,
-                                                  // i.e. the Flying, Lay-down and Multi-dimension RCs.
-    constexpr uint32_t ReverseInclineCompletedLap = (1 << 12); // Set when the vehicle travels backwards through the station for
-                                                               // the first time
-    constexpr uint32_t SpinningIsLocked = (1 << 13);           // After passing a rotation toggle track piece this will enable
-    constexpr uint32_t MoveSingleCar = (1 << 14); // OpenRCT2 Flag: Used to override UpdateMotion to move the position of
-                                                  // an individual car on a train
+    // Used on vehicles where trains can run for extended periods of time, i.e. the Flying, Lay-down and Multi-dimension RCs.
```

Comment:
```suggestion
    // Used on rides where trains can run inverted for extended periods of time, i.e. the Flying, Lay-down and Multi-dimension RCs.
```

Please fold this into the second commit. Also, please rename that commit to "Reorganise VehicleFlags comments" (making the contents clearer and using British spelling).

Reply:
Would you mind if I change it to "used on vehicles that can run inverted for extended periods of time, i.e. the Flying, Lay-down and Multi-Dimension Coaster trains" ? Since these flags are applied to vehicles, it's a little strange to say it's used on _rides_ as it was previously.